cmd/main.go

```diff
@@ -167,6 +167,9 @@ var Options struct {
 
 	// EnableSoftTimeouts is a boolean flag to enable Soft timeouts by assisted installer
 	EnableSoftTimeouts bool `envconfig:"ENABLE_SOFT_TIMEOUTS" default:"false"`
+
+	// MinVersionForNmstateService is a flag that enables the static networking flow using the nmstate service for specific OCP versions.
+	MinVersionForNmstateService string `envconfig:"MIN_VERSION_FOR_NMSTATE_SERVICE" default:"4.18"`
```

Comment:
Do we still need it in main?

Reply:
No, I forgot to delete it. I’ve just  committed the removal.